examples/stable_diffusion/inc_flux_lora_hpu.py

```diff
@@ -41,7 +41,7 @@ def is_hpu_available():
     "sdp_on_bf16": True,
     "gaudi_config": "Habana/stable-diffusion",
 }
-pipe = GaudiFluxPipeline.from_pretrained("black-forest-labs/FLUX.1-dev", torch_dtype=torch.bfloat16, **hpu_configs)
+pipe = GaudiFluxPipeline.from_pretrained("black-forest-labs/FLUX.1-dev", dtype=torch.bfloat16, **hpu_configs)
```

Comment:
I'm not 100% sure, but since `GaudiFluxPipeline` is probably based on diffusers' `FluxPipeline`, let's keep `torch_dtype` here for now.